tests/integration/plugins/test_python.py

```diff
@@ -20,12 +20,11 @@
 from pathlib import Path
 from typing import Optional
 
+import craft_parts.plugins.plugins
 import pytest
 import yaml
-from overrides import override
-
-import craft_parts.plugins.plugins
 from craft_parts import LifecycleManager, Step, errors, plugins
+from overrides import override
```

Comment:
Interesting order swap here, shouldn't third-party imports be in the middle block and project-local imports in the bottom block?

Reply:
I guess the test cases do not count as project-local imports